src/documentation/tooling/evaluation.md

```diff
@@ -0,0 +1,292 @@
+
+---
+
+Requirements for single tool documentation software (cf. [documentation tooling section](./index.md)):
+- (1) Sustainability
+- (2) Single tool toolchain
+- (3) Usability  
+    - (3a) Human-readable source format  
+    - (3b) Javadoc integration  
+    - (3c) Continuous integration capabilities  
+    - (3d) Maintainability
+    - (3e) Maintainability (dependencies)
+    - (3f) Usability of representations
+- (4) Different representation forms  
+
+---
+
+To evaluate which documentation tool may be the most suitable for our needs, we
+have marked it with a value from 1-5 for each requirement, with 1 being the
+lowest ("worst") mark and 5 the highest ("best").
+
+
+|     Tool     | 1 | 3a | 3b | 3c | 3d | 3e | 3f | x̄(3)  | 4 |
+|--------------|---|----|----|----|----|----|----|-------|---|
+| Sphinx (rST) | 5 | 3  | 1  | 3  | 4  | 3  | 4  |  3.0  | 4 |
+| Sphinx (CM)  | 3 | 5  | 1  | 3  | 2  | 3  | 4  |  3.0  | 4 |
+| Asciidoctor  | 3 | 4  | 1  | 4  | 3  | 3  | 3  |  3.0  | 3 |
+| mkDocs       | 3 | 5  | 1  | 3  | 4  | 3  | 2  |  3.0  | 1 |
+| mdBook       | 4 | 5  | 1  | 3  | 4  | 4  | 5  |  3.67 | 3 |
+| Jekyll       | 4 | 5  | 1  | 3  | 4  | 2  | 2  |  2.83 | 2 |
+
+
+### Sustainability
+
+As of now, reliable measures for predicting the sustainability of a software do 
+not exist [1], and intrinsically, the actual sustainability of a software can
+only be determined in hindsight. Therefore, assessing the sustainability
+potential for a software is a qualitative process, partly driven by the
+requirements of the project for which the software is assessed.
+
+Despite this constraint, assumptions over the sustainability potential of a
+documentation software project can be based on some assessable factors, e.g., 
+age of the software, approximated user base, development status, frequency of 
+contributions, architecture, pervasiveness of a technological community,
+level of documentation, maturity of its dependencies, etc.
+
+In the evaluation process, we have tried to approximate each candidate's
+potential for sustainability, which we present in the following.
+Additionally, we give the [SourceRank metric for high quality packages]() used by libraries.io, based on the actual products we would employ.
+
+#### Sphinx <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i>
+
+[Sphinx](http://www.sphinx-doc.org/) is a documentation generator written in
+Python. It is used to generate the [documentation for the Python programming
+language](https://docs.python.org/) itself as well as many large 
+[Python projects](http://www.sphinx-doc.org/en/master/examples.html). The
+documentation platform [Read the Docs](https://readthedocs.org/) uses Sphinx to
+automate the creation and deployment of software documentation.
+
+We have rated the sustainability of Sphinx as very high (5).  
+TODO Sphinx CM only 3  
```

Comment:
TODO?

Reply:
Fixed in 9b9ae57b456313b0c4e545ce05cc836fc91c9c48.

Reply:
Or in e88d3fad42d9d1d9c351b0a8ca86fec1a46ab35a rather.